src/components/DocumentationTopic.vue

```diff
@@ -80,10 +81,14 @@
           :class="{ 'no-primary-content': !hasPrimaryContent && enhanceBackground }"
         >
           <div
-            v-if="hasPrimaryContent"
+            v-if="hasPrimaryContent || hasOtherDeclarations"
             :class="['container', { 'minimized-container': enableMinimized }]"
           >
-            <div class="description" :class="{ 'after-enhanced-hero': enhanceBackground }">
+            <div
+              class="description"
+              :class="{ 'after-enhanced-hero': enhanceBackground }"
+              v-if="!declListExpanded"
+            >
```

Comment:
i'd put the `v-if` on top, as per Vue guidelines

```suggestion
              v-if="!declListExpanded"
              class="description"
              :class="{ 'after-enhanced-hero': enhanceBackground }"
            >
```